rest_framework/compat.py

```diff
@@ -161,10 +161,16 @@ def is_guardian_installed():
 try:
     import markdown
 
-    if markdown.version <= '2.2':
+    if hasattr(markdown, 'version'):
+        markdown_version = markdown.version
+    else:
+        # >= 3.0
+        markdown_version = markdown.__version__
```

Comment:
Here, in future, if they will remove the  `__version__` you will get exactly the same problem that you are fixing.

Reply:
Great question, I was wondering the same.

If we go for the second one (no markdown), then raising a warning could be an option (i.e. I am not using markdown not because I have not found it, but because I don't know how to handle this version).